README.md

```diff
@@ -1,2 +1,156 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+
+# SnapScan <a name="about-project"></a>
+
 SnapScan is a mobile application specially crafted to let the user have a count of their expanses. It provides the user with security and an impressive UI/UX. This app is built with Ruby On Rails.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Use database migrations**
+- **Tranction Scaning**
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/IsmailChangezi/snap-scan.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rails rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Video Explanation
+This is the link to the video explaination
+
+https://www.loom.com/share/My-Project-Journey-0daf09dd9d7242559ce9247b87e25aa8?sid=058c64bf-1f3c-4da9-85a3-62b53dcca18e
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<details>
+  <summary>Test Account</summary>
+  -email: test@gmail.com
+  -password: test123
+</details>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ismail Changezi**
+
+- GitHub: [@Ismail](https://github.com/IsmailChangezi)
+- Twitter: [@Linkeding](https://www.linkedin.com/in/ismailchangezi/)
+- 
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Create apiv1 end-point**
+- [ ] **Use OAuth**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/PrantoshB/Food-On-Rails/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a start to support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this great learning experience !
+
```

Comment:
- [ ] It is required that he README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).